c_emulator/riscv_sim.cpp

```diff
@@ -82,6 +82,8 @@ bool config_enable_rvfi = false;
 struct timeval init_start, init_end, run_end;
 uint64_t total_insns = 0;
 uint64_t insn_limit = 0;
+std::optional<uint64_t> rng_seed;
```

Comment:
Could just declare this as extern in one of the two files and then there is no need for the setter and having two duplicate variables?

Reply:
This PR is kind of subsumed by #1274 btw...